gcsio/pom.xml

```diff
@@ -97,6 +97,10 @@
       <groupId>io.grpc</groupId>
       <artifactId>grpc-protobuf</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-storage</artifactId>
+    </dependency>
```

Comment:
After adding this dependency you need to make sure that all jars that it brings are shaded.

Reply:
Thanks. Added it in shading.